### PR TITLE
Bump deps, minor cleanup

### PR DESCRIPTION
**Pull request by AlekSi**
## Readiness checklist

* [ ] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [ ] I added/updated comments and checked rendering.
* [x] I made spot refactorings.
* [ ] I updated user documentation.
* [x] I ran `task all`, and it passed.
* [x] I ensured that PR title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -46,16 +46,12 @@ body:
           - **OS**: Ubuntu 22.04
           - **Architecture**: x86_64
           - **Version**: v0.9.0
-          - **Go version**: 1.20.1
-          - **Git version**: 2.39.2
```
Comment (AlekSi) [inline .github/ISSUE_TEMPLATE/bug.yml:50]: We don't expect _users_ to know Go version or use git

File: .gitignore [hunk 2]
```diff
@@ -19,6 +19,3 @@ vendor/
 state.json
 tmp/
 website/build/
-
-# for local Docker Compse changes; see https://docs.docker.com/compose/extends/#multiple-compose-files
-docker-compose.override.yml
```
Comment (AlekSi) [inline .gitignore:24]: It is not used now

File: internal/handlers/tigris/tjson/datetime.go [hunk 3]
```diff
@@ -51,14 +51,14 @@ func (dt *dateTimeType) UnmarshalJSON(data []byte) error {
 		return lazyerrors.Error(err)
 	}
 
-	*dt = dateTimeType(time.UnixMilli(o.UnixMilli()).UTC())
```
Comment (AlekSi) [inline internal/handlers/tigris/tjson/datetime.go:54]: That was a fun way to truncate time 😬

File: internal/types/object_id.go [hunk 4]
```diff
@@ -59,8 +60,10 @@ var (
 )
 
 func init() {
-	// TODO remove for Go 1.20
-	rand.Seed(time.Now().UnixNano())
```
Comment (AlekSi) [inline internal/types/object_id.go:63]: That function is deprecated and RNG is seeded by default

File: internal/types/object_id.go [hunk 5]
```diff
@@ -59,8 +60,10 @@ var (
...
+	// to make debugging easier
+	if debugbuild.Enabled {
+		return
+	}
```
Comment (AlekSi) [inline internal/types/object_id.go:66]: So ObjectIDs are less random

**Review by w84thesun — APPROVED** [review]
👍

File: internal/handlers/tigris/tjson/datetime.go (hunk 3 above)
Comment (chilagrow) [reply internal/handlers/tigris/tjson/datetime.go:54]: Creative though!

**Review by chilagrow — APPROVED** [review]

**Pull request merged**